mm/kfence.c

```diff
@@ -103,6 +107,20 @@ static void __meminit kfence_protect(unsigned long addr)
 	__flush_tlb_one_kernel(addr);
 }
 
+static void __meminit kfence_unprotect(unsigned long addr)
+{
+	unsigned long addr_end;
+	pte_t *pte;
+	unsigned int level;
+
+	addr_end = addr + PAGE_SIZE;
+	pte = lookup_address(addr, &level);
+	BUG_ON(!pte);
```

Comment:
We'll probably have to get rid of the BUG_ONs eventually. checkpatch will remind us.

Reply:
We probably need to bail out early on syzbot and during testing, but keep the kernel working in production.
I'll replace these checks with a custom assertion that we can later make configureable.

Reply:
A if (WARN_ON_ONCE(..)) and some error-recovery path should be enough. On syzbot with panic_on_warn that'll be what we want.